pueblo/acct/timeline/check_oncall_rules.py

```
@@ -0,0 +1,327 @@
+This standalone script analyzes on-call shifts data from a CSV file
+and verifies compliance with the following rules:
+
+1. Maximum 10 on-call shifts per month (max 168 hours per month)
+2. Maximum 30 on-call days in any three-month period
+
+The script generates a report that highlights any violations and
+provides a summary of compliance status for each user.
+"""
+
+from datetime import datetime, timedelta
+from pathlib import Path
```

Ruff (0.8.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)